### PR TITLE
[RHACS] Added Jira integration topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Preview: https://openshift-docs-6h3nh0r0a-gnelson.vercel.app/openshift-acs/master/integration/integrate-with-jira.html

### CODE REVIEW

File: modules/troubleshoot-jira-integration.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+If you do not know the custom priority values in your JIRA project, you can use the `roxctl` CLI to enable debug logging for JIRA integration.
+
+.Procedure
+. To get the custom priority values from your JIRA project, run the following command to turn on debug logging for JIRA integration:
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug log --level Debug --modules notifiers/jira
+----
+. Follow the instructions to configure {product-title} for Jira integration. Now when you test the integration, even if the integration test fails, the generated log includes your JIRA project schema and the custom priorities.
```
Comment (sagidlow) [inline modules/troubleshoot-jira-integration.adoc:21]:
```suggestion
. Follow the instructions to configure {product-title} for Jira integration. When you test the integration, even if the integration test fails, the generated log includes your JIRA project schema and the custom priorities.
```

File: modules/troubleshoot-jira-integration.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+. Run the following command to save the debugging information as a compressed `.zip` file.
```
Comment (sagidlow) [inline modules/troubleshoot-jira-integration.adoc:22]:
```suggestion
. To save the debugging information as a compressed `.zip` file, run the following command:
```

File: modules/troubleshoot-jira-integration.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug dump
+----
+. Unzip the `.zip` file to retrieve the custom priority values in use in your JIRA project.
+. Turn off debug logging:
```
Comment (sagidlow) [inline modules/troubleshoot-jira-integration.adoc:29]:
```suggestion
. To turn off debug logging, run the following command:
```

Maintain consistency with above.

**Review by sagidlow — COMMENTED** [review]
Let me know if you have any questions. :)

File: modules/configure-custom-priorities-in-jira.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-jira.adoc
+:_module-type: PROCEDURE
+[id="configure-custom-priorities-in-jira_{context}"]
+= Configuring custom priorities in Jira
+
+If you are using custom priorities in your Jira project, you can configure them in {product-title}.
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/configure-custom-priorities-in-jira.adoc:8]: That changes the meaning. It basically means that if users have defined custom priorities in Jira, such as `cool`, `supercool`, `hot`, and `superhot` They need to follow the steps here to configure it in RHACS.

**Comment by gaurav-nelson** [comment]
Received LGTM from @sagidlow , thank you for reviewing.

**Pull request merged**